rest/src/main/kotlin/builder/interaction/OptionsBuilder.kt

```diff
@@ -174,10 +174,38 @@ public class NumberOptionBuilder(name: String, description: String) :
 public class StringChoiceBuilder(name: String, description: String) :
     BaseChoiceBuilder<String>(name, description, ApplicationCommandOptionType.String) {
 
+    private var _minLength: Optional<Int> = Optional.Missing()
+
+    /**
+     * The minimum string length (minimum of 0).
+     */
+    public var minLength: Int? by ::_minLength.delegate()
+
+    private var _maxLength: Optional<Int> = Optional.Missing()
+
+    /**
+     * The maximum string length (minimum of 1).
```

Comment:
```suggestion
     * The maximum allowed string length (minimum of `1`, maximum of `6000`).
```
see https://github.com/discord/discord-api-docs/commit/5981d3fa3893a7f45fc7272d7a40796d13ad03b6